main.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAABFwAAAAdz
+c2gtcnNhAAAAAwEAAQAAAQEAh4jBgn2dumeHJ+YEKid4PKlMKtcZkgS+BPQ3teoY
+hmcF1j3PeO+HlWLzibJy9036+zWOMdR9LP3YSc+6i555+sj7fxZOroThaqpPHo0g
+SgJDwQlxYAgKleR2kOv1P0EUSYY6yE/7nfWYy7gmJHksGG91xl4rRzpljGgFMYtA
+OVwv9PAThvUTO2Jl6ytB1mng7axZC8evcFn/nWNGMET6h/nKHv9qdLaP9s+OvryR
+KKWqoWKbhNYBVFEbHkmfAL5c0WZ7TpfIsMx0Nz8cfez6hk+s4kwaE1cMe47p46Po
+7lktSHfqpe1azNJG/Iv9QY2Pxju49Eb+4qoFNtF/wYMS5QAAA8D0qNxI9KjcSAAA
+AAdzc2gtcnNhAAABAQCHiMGCfZ26Z4cn5gQqJ3g8qUwq1xmSBL4E9De16hiGZwXW
+Pc9474eVYvOJsnL3Tfr7NY4x1H0s/dhJz7qLnnn6yPt/Fk6uhOFqqk8ejSBKAkPB
+CXFgCAqV5HaQ6/U/QRRJhjrIT/ud9ZjLuCYkeSwYb3XGXitHOmWMaAUxi0A5XC/0
+8BOG9RM7YmXrK0HWaeDtrFkLx69wWf+dY0YwRPqH+coe/2p0to/2z46+vJEopaqh
+YpuE1gFUURseSZ8AvlzRZntOl8iwzHQ3Pxx97PqGT6ziTBoTVwx7junjo+juWS1I
+d+ql7VrM0kb8i/1BjY/GO7j0Rv7iqgU20X/BgxLlAAAAAwEAAQAAAQBEXzvnfUEl
+ePaYrROqxFCGoY04C1zt6ZCGZUD5d0e5/ZzXMEVKp+NI/CsJaslOcrtLIQBDqQMg
+wEntKmrb5i140wz0VqeIqKYirhnNurnwO+Y8KmxUWpokdaVOhIoqx8TZNmrPBHJV
+hED5fENZoPsatJfAY18HxrDVvN/p12qlxFoZPiTmFi/lY1tJ6+FdLQ+Zn4PW/Vc7
+iYCrQwm6TngGXjBHmHI9nEB+UyS8V8VH8dobLUsxqHCJxzMbFxYHRxlWtbpdKj8n
+F0xqeqftwhMkkoyi3EBkczDSwMv29SxhPTpQkszv2J3XcIy0fIwE0SXlDt68dPFA
+w/ydkg2Tk1kdAAAAgF+peavBXAs4NgwtfbKX7BlmIVD76qX9pjsQFJvMdcY8NsoI
+ht8BbbmUGwTYeJgn8lljknOgTfTNbYE38bBA/1M+pooQfw/pNISKI7Zsux+q4vn+
+eGYyWE9KH8OI3wi/08eaMBZQtBNJH6IAKiAG3wrc/nHI8uBxSfZELw9l/6CXAAAA
+gQDcjOxZvrQZ3uR4kgIfmFk2Gzt0FzvfG6gpKyOD08JApxOfwGJbAeonTGM9RrY0
+ZpswDwGI0ncmmWvxA8RFsrbmuEEcl26eZlI7xpRWDzbUE9nAzDJb0ZYWGLjvmy4W
+js4SZ/c5qDvVhMQ3OcJlJ9hWiuqE/Npl7ndTuLbR00XcgwAAAIEAnVGieCAVwFXg
+kDXNuuaCPUvI5pPS20Poyj6mdUw5nJRLGAr/A43pXC/VqDh4037QGhknwySSLNx0
+3WWHRKyG5SlMQ4mWQ+sjjc+xHRt6qs4wkG8USjvvHi/Xd3B1+1T2HNJyoiZfTB7J
+vdww8IaRsx/id9tdeCnXa6jXg2JfhncAAAAEbWFpbgECAwQFBgc=
+-----END OPENSSH PRIVATE KEY-----
```

Comment:
could you please delete your private key as it is not a good security practice to have it here.